app/Http/Controllers/Settings/NativeTrackingController.php

```
@@ -0,0 +1,68 @@
+     * current workspace and 404s on a foreign or missing id.
+     */
+    public function store(Request $request, ConnectedAccount $account): RedirectResponse
+    {
+        /** @var User $user */
+        $user = $request->user();
+        $workspace = $user->currentWorkspace;
+        abort_if($workspace === null, 404);
+        $this->authorizeManage($user, $workspace->id);
+
+        if (! $account->platform->supportsNativeRead()) {
+            throw ValidationException::withMessages(['account' => 'This platform does not support native tracking.']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Validation\ValidationException' in method 'store'. (undefined)

(StaticAccess)